docs/user-guide/home-page.md

```diff
@@ -0,0 +1,78 @@
+---
+sidebar-position: 2
+---
+
+# Home page
+
+To get started with Resuminator, you will be greeted with our Home page, which shows you the following screen:
+
+![Home Page](/doc_img/home_page.png)
+
+The Home page can be divided into the following various parts to understand it better ~
+
+## Side-Bar
+
+### Workspace
+
+- **Settings**: Settings tab routes you to your account settings which contain the general, account and advanced tabs.
+
+### Resources
+
+- **Broadmaps**: Broadmaps was a 36 issue weekly newsletter with high-signal information on products, tech, and growth
+  ⚡ Curated with ♥ by [@\_viveknigam\_](https://github.com/viveknigam3003) &
+  [@\_mercurybuddy](https://github.com/merrcury)
+
+- **Help & Support**: This tab routes you to the [Help & Support](https://docs.resuminator.in/docs/support/)
+  page on the Resuminator Docs website for know-how of our product.
+
+- **Discord Server**: Join our [Discord](https://discord.com/invite/uJRFbDHHAJ)
+  for support, feedback and sneak-peaks in our alpha-program.
+
+## Main Space
+
+This the main area where you can add and delete resumes and edit their name / icon by clicking on the name.
+
+### From Scratch
+
+> If you are creating a resume for the first time, you should refer to our
+> [Quick Start](/introduction/quick-start.md) guide.
+
+### Existing
+
+If you have created at least one resume, then you can also take advantage of the template feature.
+
+![Template Feature](/doc_img/template_feat.png)
+
+:::tip
+Template Feature can be used when you want to perform minor tweaks in your resume according to your job
+roles when applying for multiple companies.
+:::
+
+## Navigation
+
+### Navigation-links
+
+- **Home**: The place where you create and manage your resumes.
+
+- **Create**: This is where the actual magic happens and where you cook your resume with our
+  drag-and-drop and easy-to-use technology.
+
+:::caution
```

Comment:
Please intend this caution block and make `Create` bold in it.

Reply:
can you please explain "intend this caution block" as I'm not getting what I have to do here
did you mean indent?

Reply:
Ya sorry, typo on my end. The `indent` was my intention.